compiler/rustc_privacy/src/errors.rs

```diff
@@ -87,5 +90,13 @@ pub struct FromPrivateDependencyInPublicInterface<'a> {
 pub struct PrivateInPublicLint<'a> {
     pub vis_descr: &'static str,
     pub kind: &'a str,
-    pub descr: String,
+    pub descr: FromDisplay<'a>,
+}
+
+pub struct FromDisplay<'a>(pub &'a dyn Display);
```

Comment:
Do you think it would make sense to have an `Into` impl for anything that implements `Display`, and doing `field.into()` rather than using the constructor?

Reply:
That makes sense, yeah